api/src/main/java/org/openmrs/module/commonreports/CommonReportsConstants.java

```diff
@@ -33,6 +33,8 @@
 	
 	public static final String COMPONENT_REPORTMANAGER_LAB = MODULE_ARTIFACT_ID + ".lab";
 	
+	public static final String COMPONENT_REPORTMANAGER_FAMILYPLANNING = MODULE_ARTIFACT_ID + ".familyPlanning";
```

Comment:
@reagan-meant, are you using this constant?

Reply:
Removed